src/BuildTools.sln

```diff
@@ -43,8 +43,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.DotNet.CodeAnalys
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "xunit.console.uwp", "xunit.console.uwp\xunit.console.uwp.csproj", "{36219AA4-D36A-4CB4-9C7B-FB48641D411A}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "xunit.runner.uwp", "xunit.runner.uwp\xunit.runner.uwp.csproj", "{BB960A1D-DE0B-43F5-ACA1-0BDEA8E63CDD}"
```

Comment:
@AlexGhiondea  removed this project, was causing an annoying icon in my solution explorer.